source/test/test_datastore.py

```diff
@@ -0,0 +1,100 @@
+import sys
+sys.path.append("./lambda/helper/python")
+import boto3
+import unittest
+from moto import mock_s3
+from moto import mock_dynamodb2
+import datastore
+
+DOCUMENTS_TABLE_NAME="DocumentsTestTable"
+OUTPUT_TABLE_NAME="OutputTestTable"
+REGION="us-west-2"
+
+@mock_dynamodb2
+class TestDocumentStore(unittest.TestCase):
+    def setUp(self):
+        self.conn = boto3.client('dynamodb',region_name=REGION)
+        self.conn.create_table(
+            TableName = DOCUMENTS_TABLE_NAME,
+            KeySchema = [{"AttributeName": "documentId","KeyType":"HASH"}],
+            AttributeDefinitions=[{"AttributeName": "documentId", "AttributeType": "S"}],
+            ProvisionedThroughput={"ReadCapacityUnits": 5, "WriteCapacityUnits": 5},
+        )
+        self.conn.put_item(
+            TableName = DOCUMENTS_TABLE_NAME,
+            Item={
+                "documentId": {"S" : "b1a54fda-1809-49d7-8f19-0d1688eb65b9"},
+                "objectName": {"S": "public/samples/Misc/expense.png"},
+                "bucketName": {"S": "dusstack-sample-s3-bucket"},
+                "documentStatus": {"S": "IN_PROGRESS"}
+            }
+        )
+        self.conn.put_item(
+            TableName = DOCUMENTS_TABLE_NAME,
+            Item={
+                "documentId": {"S" : "b1a99fda-1809-49d7-8f19-0d1688eb65b9"},
+                "objectName": {"S": "public/samples/Misc/expense.png"},
+                "bucketName": {"S": "dusstack-sample-s3-bucket"},
+                "documentStatus": {"S": "IN_PROGRESS"}
+            }
+        )
+        self.ds = datastore.DocumentStore(DOCUMENTS_TABLE_NAME,OUTPUT_TABLE_NAME)
+    
+    def test_create_document_success(self):
+        bucketName = "dusstack-sample-s3-bucket"
+        objectName = "public/samples/Finance/report.pdf"
+        documentId = "b1a66fda-1809-49d7-8f19-0d1688eb65b9"
+        response = self.ds.createDocument(documentId, bucketName, objectName)
+        self.assertEqual(response, None)
+    
+    def test_create_duplicate_documentid_throws_error(self):
+        bucketName = "dusstack-sample-s3-bucket"
+        objectName = "public/samples/Finance/report.pdf"
+        documentId = "b1a54fda-1809-49d7-8f19-0d1688eb65b9"
+        response = self.ds.createDocument(documentId, bucketName, objectName)
+        self.assertEqual(response, {'Error': 'Document already exist.'})
+    
+    def test_update_document_status_success(self):
+        documentId = "b1a54fda-1809-49d7-8f19-0d1688eb65b9"
+        response = self.ds.updateDocumentStatus(documentId, "FAILED")
+        self.assertEqual(response, None)
+    
+    def test_update_document_status_throws_error_when_document_doesnot_exist(self):
```

Comment:
Can we can a underscore between all words? *does_not*

Reply:
Will update